### ISSUE TITLE
Waypoint: Create a JavaScript Slot Machine - Incorrect formulat

### ISSUE THREAD

**Issue by AlexFeng217** [FreeCodeCamp/FreeCodeCamp#4009]
Math.floor(Math.random() * (3 - 1 + 1)) + 1 is the same as Math.floor(Math.random() * (3)) + 1 

It is unnecessary to subtract 1 then add 1 inside the parenthesis. This may cause confusion for beginners not understanding why you must subtract 1 then add one. In both cases the number being multiplied against Math.random() will evaluate to 3. 

3-1+1 will always evaluate to 3.

**Comment by awesomeaniruddh** [comment]
The reasoning behind the formula is explained in an earlier waypoint: http://www.freecodecamp.com/challenges/waypoint-generate-random-whole-numbers-within-a-range

The same waypoint also links to an explanation of the formula in the Wiki 
(https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Waypoint-Generate-Random-Whole-Numbers-within-a-Range#explanation).

Removing the 1s from the formula will confuse campers. Anyone any thoughts?

**Comment by SaintPeter** [comment]
I agree with @AlexFeng217 - it makes sense to use this formula when explaining how to generate random numbers, but I'm not sure that it continues to make sense in the context of this problem.  We have issues with campers who are manually typing the code (rather than copying/pasting) missing a sign or a space and it breaking for non-obvious reasons.

I had been considering making other changes to this set of waypoints anyway, I'll see if it makes sense to include this as well.
